packages/did-core-test-server/suites/methods/did-key/did-key.spec.js

```diff
@@ -0,0 +1,27 @@
+let { suiteConfig } = global;
```

Comment:
This is about the name of the file... probably want did-key-transmute.spec.js -- as DB is going to want to submit their own as well.

Reply:
@msporny yes, this part of writing tests remains undefined.... IMO, the method conformance tests should ideally be a single collection, and represent all legal examples produced by real world implementations, for did-key, the problem is that implementations don't support all representations.... this can be solved by splitting the did-key by implementation, which appears to be in line with what @peacekeeper  wants.